src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -101,10 +101,8 @@ public StudentAttributes(CourseStudent student) {
         this.lastName = student.getLastName();
         this.comments = Sanitizer.sanitizeTextField(student.getComments());
         this.team = student.getTeamName();
-        this.section = student.getSectionName() == null ? Const.DEFAULT_SECTION
-                                                        : student.getSectionName();
-        this.googleId = student.getGoogleId() == null ? ""
-                                                      : student.getGoogleId();
+        this.section = student.getSectionName() == null ? Const.DEFAULT_SECTION : student.getSectionName();
```

Comment:
It's one line so there's nothing to align. Also, this is not Eclipse but my doing, just happened to spot it :p